CLUtils/CogSLanguageUtilities.Core/Services/Parser/DocxParserService.cs

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I think this starting to get out of scope of this CLI tool. This should be maintained by another team and you just consume it, but that's just my opinion.

Reply:
It’s in the scenarios we address so the parsing is part of it
We can have it independent as a tool but that is not now for sure